### ISSUE TITLE
Another Docker distro keeping Redis?

### ISSUE THREAD

**Issue by timreuscher** [zedeus/nitter#510]
Any chance you'd consider maybe another branch (nitter-redis?) in the Docker project to add Redis back in? I've been using Nitter locally on my Synology NAS successfully until you split it out. Have had zero success since, even with docker-compose. It simply won't find Redis no matter what I try.

**Comment by LainLayer** [comment]
We wont be creating another branch for this, but maybe someone can help you out with your problem.
Would help if you provide as much possibly relevant details as possible.

**Comment by timreuscher** [comment]
No matter what I put in the nitter.conf file it wouldn't see Redis. But I did figure it out. I had to use 
> redisHost = "172.19.0.2"
in order for it to work properly. Obviously that's the internal Docker IP for Redis. Not sure if this is common, but Synology's Docker implementation is known for screwy networking. That probably means it's not going to be stable after a reboot, but I guess I can live with that.
